app/styles/base.css

```diff
@@ -582,6 +582,9 @@ select:active {
 #noVNC_setting_path {
   width: 100px;
 }
+.noVNC_disabled_setting {
```

Comment:
Sure, I can't find any good place to put it in the css though. Perhaps just at the top below .noVNC_only_touch.noVNC_hidden?